website/docs/quick-start-guide.md

```diff
@@ -175,18 +175,163 @@ values={[
 </TabItem>
 <TabItem value="sparksql">
 
+Spark-sql needs an explicit create table command.
+
+- Table types:
+  Both types of hudi tables (CopyOnWrite (COW) and MergeOnRead (MOR)) can be created using spark-sql.
+
+  While creating the table, table type can be specified using **type** option. **type = 'cow'** represents COW table, while **type = 'mor'** represents MOR table.
+
+- Partitioned & Non-Partitioned table:
+  Users can create a partitioned table or non-partitioned table in spark-sql.
+  To create a partitioned table, one needs to use **partitioned by** statement to specify the partition columns to create a partitioned table.
+  When there is no **partitioned by** statement with create table command, table is considered to be a non-partitioned table.
+
+- Managed & External table:
+  In general, spark-sql supports two kinds of tables, namely managed and external.
+  If one specifies a location using **location** statement or use `create external table` to create table explicitly, it is an external table, else its considered a managed table.
+  You can read more about external vs managed tables [here](https://sparkbyexamples.com/apache-hive/difference-between-hive-internal-tables-and-external-tables/).
+
+- Table with primary key:
+  Users can choose to create a table with primary key as required. Else table is considered a non-primary keyed table.
```

Comment:
Shall we mention here non-pk table not supported? and explain how `uuid` works as the implicit default pk

Reply:
ok, i'll update here

Reply:
i'll remove `Table Types` and `Table with primary key` sections which are redundant with `Create Table Properties` below. And move `Create Table Properties` and notes here.